.gitlab/build_lassen.yml

```diff
@@ -2,9 +2,9 @@
 # This is the share configuration of jobs for lassen
 .on_lassen:
   variables:
-    ALLOC_COMMAND: lalloc ${ALLOC_NODES} -W ${ALLOC_TIME} -q pci -G ${ALLOC_BANK}
+    SCHEDULER_PARAMETERS: -nnodes ${ALLOC_NODES} -W ${ALLOC_TIME} -q pci -G ${ALLOC_BANK}
   tags:
-    - shell
+    - batch
```

Comment:
What is a batch runner? Is there a confluence page on this?

Reply:
There unfortunately is not. Basically a batch runner is a shell runner that takes care of scheduling for you. I honestly don't know all the things it does for us but LC suggested it would be better.